run_parallel_memverse_tests.py

```
@@ -0,0 +1,159 @@
+            devices = subprocess.check_output(["adb", "devices"]).decode()
+            if any('emulator-' in l and 'device' in l for l in devices.splitlines()):
+                # Confirm sys.boot_completed
+                try:
+                    bc = subprocess.check_output([
+                        "adb", "shell", "getprop", "sys.boot_completed"
+                    ]).decode().strip()
+                    if bc == '1':
+                        booted = True
+                        print("Emulator booted!")
+                        break
+                except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)